common/changes/@uifabric/example-app-base/jspurlin-ComboBoxScrollIntoViewFix_--2090-_2017-06-29-13-27.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
```

Comment:
<shrug> That's what "rush change" spit out even though I just pressed ENTER to advance past the info for the other packages (which is what it says to do)